### PR TITLE
Links tokens in InputLookupQA component.

### PR DESCRIPTION
**Pull request by justinlittman**
refs #971

### CODE REVIEW

File: src/components/editor/property/InputLookupQA.jsx [hunk 1]
```diff
@@ -12,6 +14,7 @@ import { itemsForProperty, getDisplayValidations, getPropertyTemplate } from 'se
 import { changeSelections } from 'actions/index'
 import { booleanPropertyFromTemplate, getLookupConfigItems } from 'Utilities'
 import Config from 'Config'
+// import {getOptionLabel} from 'react-bootstrap-typeahead/utils';
```
Comment (jcoyne) [inline src/components/editor/property/InputLookupQA.jsx]: Cruft?

File: src/components/editor/property/InputLookupQA.jsx [hunk 2]
```diff
@@ -218,6 +235,7 @@ class InputLookupQA extends Component {
     )
   }
 }
+// {getOptionLabel(option, props.labelKey)}
```
Comment (jcoyne) [inline src/components/editor/property/InputLookupQA.jsx]: cruft?

**Comment by justinlittman** [comment]
Grrr ... I wish I could get GH to prompt me "Have you checked for cruft?" before creating a PR.

@jcoyne  - decrufted.

**Pull request merged**